### DISCUSSION TITLE
plot_pca_correlation_graph Font size change

### DISCUSSION THREAD

**Discussion by CasperJaa** [rasbt/mlxtend#973] [Q&A]
Hi, I've recently used the plot_pca_correlation_graph but I haven't found a way to change the font size of the attribute names and the graph is relatively difficult to read currently.

**Comment by NimaSarajpoor** [comment]
@CasperJaa 
This should work:
```
# example taken from: http://rasbt.github.io/mlxtend/user_guide/plotting/plot_pca_correlation_graph/

from mlxtend.data import iris_data
from mlxtend.plotting import plot_pca_correlation_graph
import numpy as np

X, y = iris_data()

X_norm = X / X.std(axis=0) # Normalizing the feature columns is recommended

feature_names = [
  'sepal length',
  'sepal width',
  'petal length',
  'petal width']

my_fig_obj, my_correlation_matrix = plot_pca_correlation_graph(X_norm, 
                                                        feature_names,
                                                        dimensions=(1, 2),
                                                        figure_axis_size=10)
```

Now, you need to get the axes from the object `my_fig_obj`, and change it:

```
from matplotlib import pyplot as plt

# getting axes
ax_list = my_fig_obj.axes # alternatively, my_fig_obj.gca() may work as well 
ax = ax_list[0] # you can do print(ax_list) to see what it contains. 

# change label and font
ax.set_xlabel('hello', fontsize=15)
ax.set_ylabel('world', fontsize=25)

# plot again
my_fig_obj.figure
```

This is what I got:

![my_fig](https://user-images.githubusercontent.com/38519522/191773177-046c4217-f786-4d9d-90d4-c082cc384221.jpeg)


---

Also,  you may want to wait for @rasbt response. He knows better :)

**Reply by NimaSarajpoor** [reply]
@CasperJaa 
Oops... I think misread the question... You were asking for the attributes of the data (for instance, `sepal width` in the figure above) 

Right? 

(I do not know how  to do this)

**Reply by CasperJaa** [reply]
Yes, those are the ones I would like to change. Thanks for the answer anyway. You'll never know when it may come in handy

**Comment by rasbt** [comment]
The code for this is quite involved, and it would actually be quite tricky to do that in matplotlib after calling the function (maybe impossible). I would suggest to adjust the size of the text in post processing via a vector graphics editor. Or, you can create a custom version of the plotting function; I think this is the line that you'd need to change: https://github.com/rasbt/mlxtend/blob/master/mlxtend/plotting/pca_correlation_graph.py#L172
